ddtrace/profiling/profiler.py

```diff
@@ -160,15 +160,15 @@ def _build_default_exporters(self):
                 endpoint = agent.get_trace_url()
 
         if self.agentless:
-            endpoint_path = "/v1/input"
+            endpoint_path = "/api/v2/profile"
```

Comment:
Does this require a minimum agent version? Any chance this could be a breaking change if we no longer support v1?

Reply:
This is specifically for agentless so I think this doesn't care about agent version.

Reply:
ah right 🤦

Reply:
So I'm not sure this is needed. The point of this PR is to support the new upload format, including appropriate new endpoint for agentless.